src/Compilers/CSharp/Portable/Symbols/MethodSymbolExtensions.cs

```diff
@@ -225,5 +220,15 @@ internal static bool IsValidUnscopedRefAttributeTarget(this MethodSymbol method)
                 !method.IsConstructor() &&
                 !method.IsInitOnly;
         }
+
+        internal static bool HasUnscopedRefAttributeOnMethodOrProperty(this MethodSymbol? method)
+        {
+            if (method is null)
+            {
+                return false;
+            }
+            return method.HasUnscopedRefAttribute ||
+                method.AssociatedSymbol is PropertySymbol { HasUnscopedRefAttribute: true };
```

Comment:
>method.AssociatedSymbol is PropertySymbol { HasUnscopedRefAttribute: true }; [](http://example.com/codeflow?start=16&length=76)

What about event accessors?

Reply:
> What about event accessors?

There was no response to this question.

Reply:
Logged [https://github.com/dotnet/roslyn/issues/66274](https://github.com/dotnet/roslyn/issues/66274) to follow up in a subsequent PR.